### ISSUE TITLE
[Bug] Custom template doesn't load

### ISSUE THREAD

**Issue by sonomirco** [dotnet/docfx#10378]
**Describe the bug**
Just tried this simple step.
https://dotnet.github.io/docfx/docs/template.html?tabs=modern#custom-template

**To Reproduce**
Under the docs folder, I have created a folder called aecom-template.
Added to the docfx.json file

```
    "template": [
      "default",
      "modern",
      "aecom-template"
    ],
```
Used the main.css and main.js as documentation, but nothing happened.
I don't have the GitHub logo, and no colour is changed.

**Expected behavior**
I am expected to see the GitHub logo and the same green colour you have on your website, adding those steps.

**Context (please complete the following information):**
- OS: Windows
- Docfx version: 2.77

**Comment by mxcolin** [comment]
Same here, followed the documentation as specified and it does nothing at all, no changes are observed.

**Comment by filzrev** [comment]
I've tested on my environment. 
And it seem works as expected. (After `Ctrl+F5` browser refresh)

Is it able to test by **temporary** [disabling browse caching](https://learn.microsoft.com/en-us/answers/questions/938647/devtools-sources-caching-issue)

**Comment by mxcolin** [comment]
Yeah, that was it. Disabling the browser cache did the trick. Thanks!!!

**Comment by xv** [comment]
> I've tested on my environment. And it seem works as expected. (After `Ctrl+F5` browser refresh)

Thanks so much!! I spent all night trying to figure out why my template override wasn't working. Then I found this post, haha.

Perhaps a note/reminder to clear cache would be nice to have on the page linked by sonomirco.

**Comment by filzrev** [comment]
Browser cache issue is expected to be fixed after #10891 merged.

On currently `docfx serve` command behavior.
[browser's default heuristic](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Caching#heuristic_caching) policy is used. and it seems to be causing  old-cache issue (Especially when file's `LastModified`  date is too old).
